### PR TITLE
doc: mds cap acquisition readdir throttle documentation

### PR DESCRIPTION
**Pull request by kotreshhr**
Fixes: https://tracker.ceph.com/issues/49763
Signed-off-by: Kotresh HR <khiremat@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by kotreshhr** [comment]
jenkins render docs

File: doc/cephfs/cache-configuration.rst [hunk 1]
```diff
@@ -138,6 +138,47 @@ counters as needed to resolve any slow recall warnings in the cluster health
...
+    mds_session_cap_acquisition_throttle (default: 500K)
+
+and::
+
+    mds_session_cap_acquisition_decay_rate (default: 10 seconds)
+
+The cap acquisition decay counter controls the rate of cap acquisition via
+readdir. The behavior of the decay counter is the same as for cache trimming or
+caps recall. Each readdir call increments the counter by the number of files in
+the directory.
```
Comment (batrick) [inline doc/cephfs/cache-configuration.rst:163]:
```suggestion
caps recall. Each readdir call increments the counter by the number of files in the result.
```

**Review by batrick — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Comment by batrick** [comment]
> jenkins render docs

You don't need to do this anymore. Just look at the "Details" link for the "docs/readthedocs.org:ceph" check.

**Review by batrick — APPROVED** [review]

**Pull request merged**